onegov/swissvotes/models/vote.py

```diff
@@ -559,24 +560,67 @@ def results_cantons(self):
 
     def get_recommendation(self, name):
         """ Get the recommendations by name. """
+        # Todo: remove inconsistencies in table name
+        name_ = name
+        return self.codes('recommendation').get(
+            self.recommendations.get(name_)
+        )
 
-        recommendations = self.recommendations or {}
-        return self.codes('recommendation').get(recommendations.get(name))
+    def get_recommendation_of_existing_parties(self):
+        """ Get only the existing parties as when this vote was conducted """
+        if not self.recommendations:
+            return {}
+        return {k: v for k, v in self.recommendations.items() if v != 9999}
```

Comment:
Since `9999` is a special value that is reused below I would introduce a constant:

```python
ORGANIZATION_NO_LONGER_EXISTS = 9999
```

And then compare it against this constant, then it's clear what the intent of this number is.

Reply:
Okay. I introduced it in the top section of the class.